ligo/src/registryDAO.mligo

```diff
@@ -36,6 +71,11 @@ let require_unpacked_registry_affected (packed_b: bytes) : registry_affected =
     Some (v) -> v
   | None -> (failwith "UNPACKING_FAILED" : registry_affected)
 
+let require_unpacked_proposal_receivers (packed_b: bytes) : proposal_receivers =
+  match ((Bytes.unpack packed_b) : (proposal_receivers option)) with
+    Some (v) -> v
```

Comment:
```suggestion
  | Some (v) -> v
```

Reply:
Updated. Here and some other places where this was missing.